Kuroba/app/src/main/java/com/github/adamantcheese/chan/utils/ChanUtils.kt

```diff
@@ -0,0 +1,15 @@
+package com.github.adamantcheese.chan.utils
+
+import com.github.adamantcheese.chan.core.model.orm.Loadable
+
+object ChanUtils {
+
+    /**
+     * Extracts and converts to a string only the info that we are interested in from this loadable
+     */
+    @JvmStatic
+    fun loadableToString(loadable: Loadable): String {
```

Comment:
You've moved this from ThreadSaveManager, but it would be better if this function was internal to the Loadable object, say, `toShortString` or something of that sort.

Reply:
Yeah, that's probably where it should be.

Reply:
Done.